package.json

```diff
@@ -82,37 +82,38 @@
     "zone.js": "0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "13.3.7",
-    "@angular-eslint/eslint-plugin": "13.0.1",
-    "@angular-eslint/eslint-plugin-template": "13.0.1",
-    "@angular-eslint/template-parser": "13.0.1",
-    "@angular/cli": "13.3.7",
-    "@angular/compiler-cli": "13.3.11",
-    "@angular/language-service": "13.3.11",
+    "@angular-devkit/build-angular": "14.0.1",
+    "@angular-eslint/eslint-plugin": "13.2.1",
+    "@angular-eslint/eslint-plugin-template": "13.2.1",
+    "@angular-eslint/template-parser": "13.2.1",
+    "@angular/cli": "14.0.1",
+    "@angular/compiler-cli": "14.0.1",
+    "@angular/language-service": "14.0.1",
     "@compodoc/compodoc": "1.1.15",
     "@ngneat/transloco-keys-manager": "3.3.3",
     "@ngneat/transloco-optimize": "3.0.2",
     "@ngworker/spectacular": "0.2.0",
-    "@nrwl/cli": "14.1.9",
-    "@nrwl/cypress": "14.1.9",
-    "@nrwl/eslint-plugin-nx": "14.1.9",
-    "@nrwl/jest": "14.1.9",
-    "@nrwl/linter": "14.1.9",
-    "@nrwl/nx-cloud": "14.0.5",
-    "@nrwl/storybook": "14.1.9",
-    "@nrwl/workspace": "14.1.9",
+    "@nrwl/cli": "14.3.2",
```

Comment:
Remember to update NOTICE.md when upgrading packages :)

Reply:
Done.